controllers.js

```diff
@@ -0,0 +1,14 @@
+const { getTopics, getEndpoints} = require('./models')
+
+
+exports.fetchTopics = (req, res) => {
+    getTopics().then((result) => {
+        res.status(200).send(result)
+    })
+}
+
+exports.fetchEndpoints = (req, res) => {
```

Comment:
catch blocks on each controller please, will need a 500 error handler to catch endpoint errors we cant expect.